test/functional/rpc_help.py

```diff
@@ -25,7 +25,13 @@ def run_test(self):
 
         # command titles
         titles = [line[3:-3] for line in node.help().splitlines() if line.startswith('==')]
-        assert_equal(titles, ['Blockchain', 'Control', 'Generating', 'Mining', 'Network', 'Rawtransactions', 'Util', 'Wallet', 'Zmq'])
+
+        components = ['Blockchain', 'Control', 'Generating', 'Mining', 'Network', 'Rawtransactions', 'Util', 'Wallet']
+
+        if is_zmq_enabled(self):
```

Comment:
nit, my preference is
```py
if is_zmq_enabled(self):
    assert_equal(titles, ['Blockchain', 'Control', 'Generating', 'Mining', 'Network', 'Rawtransactions', 'Util', 'Wallet', 'Zmq'])
else:
    assert_equal(titles, ['Blockchain', 'Control', 'Generating', 'Mining', 'Network', 'Rawtransactions', 'Util', 'Wallet'])
```

Reply:
Fair enough.. I'll change it :-)

Reply:
there might be more optional categories in the future, which would result in a combination blow-up (say, if we ever make the functional tests run without wallet). I prefer adding them one by one as it is now.

Reply:
Makes sense!